packages/shared/auth0-client/src/index.ts

```diff
@@ -1,7 +1,11 @@
 import { APIResponse, errorResponse, ResponseStatus, successResponse, unhandledError } from '@weco/identity-common';
-import axios, { AxiosInstance } from 'axios';
+import axios, { AxiosError, AxiosInstance } from 'axios';
 import { Auth0Profile, Auth0UserInfo, toAuth0Profile, toAuth0UserInfo } from './auth0';
 
+function responseCodeIs(responseCode: number) {
```

Comment:
Something for `shared/identity-common` perhaps, so we can reuse it both here and in `sierra-client`?

Reply:
Yep, actually forgot to use it in my second commit. Going to update that and will move it.